apps/cowswap-frontend/src/api/gnosisProtocol/errors/OperatorError.ts

```diff
@@ -7,66 +7,86 @@ export interface ApiErrorObject {
 }
 
 // Conforms to backend API
-// https://github.com/cowprotocol/services/blob/main/crates/orderbook/openapi.yml#L801
-// and
-// https://github.com/cowprotocol/services/blob/main/crates/orderbook/openapi.yml#L740
+// https://github.com/cowprotocol/services/blob/d66ff0b0c5a187264f3b0f86810ef04da21255e8/crates/orderbook/openapi.yml#L1214-L1239
 export enum ApiErrorCodes {
+  AlreadyCancelled = 'AlreadyCancelled',
```

Comment:
feel free to ignore for this fix, but I believe there's an enum automatically created from swagger.

Also, probably we should not use this enum until the backend dynamically create the swagger docs from their code. They plan to do that, but haven't done it yet.

Reply:
Hmm, good point.
I'll try to use that if possible because I don't like this manual mapping.
There were a couple missing from the swagger list that I added now.

Reply:
This will need a fix on SDK, after services is fixed and deployed -> https://github.com/cowprotocol/services/pull/2400

Will address it afterwards

Reply:
https://github.com/cowprotocol/cowswap/issues/3834